tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -216,7 +216,7 @@ func (a *AwsProvider) createWindowsVM(usePrivateSubnet bool) (windowsVM types.Wi
 	// Build new credentials structure to be used by other actors. The actor is responsible for checking if
 	// the credentials are being generated properly. This method won't guarantee the existence of credentials
 	// if the VM is spun up.
-	// Credentials can have both public and privateIP Addresses.
+	// Credentials can have both public or privateIP Addresses.
```

Comment:
Sorry to nitpick :-) but it has to be `either public or private IP addresses`

Reply:
No problem